.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
       - auth:
           password: $DOCKER_PASSWORD
           username: $DOCKER_USERNAME
-        image: circleci/node:16.13.0
+        image: circleci/node:18.10.0
```

Comment:
Looks like the build failed. I'm wondering if we need to use the latest? Looks like we're attempting to use a deprecated image https://circleci.com/developer/images/image/cimg/node

Reply:
It looks like the deprecated circleci/node:* images don't have one for this node version based on the build failure we got https://app.circleci.com/pipelines/github/trussworks/react-uswds/9313/workflows/5d04dc51-0c15-4d6c-ae43-45911c6963d3/jobs/9348/steps

Reply:
```suggestion
        image: cimg/node:18.10.0
```

Reply:
Just to note, this new image is the one to use, but it has also removed `python` version 2.x so I had to update the deploy scripts to use `python3` instead.